nuc_morph_analysis/analyses/density/run_all_density_workflows.py

```diff
@@ -1,6 +1,9 @@
-from nuc_morph_analysis.analyses.density import watershed_validate
-watershed_validate.run_validation_and_plot(plot_everything=False)
-from nuc_morph_analysis.analyses.density import effect_of_resolution_level
-from nuc_morph_analysis.analyses.density import pseudocell_based_density_analysis
+# this code runs all workflows in the density folder (not all of these are in the paper)
```

Comment:
I dont think you have to say this now that we have the figure numbered workflow. But also it begs the question which ones are in the paper?

Maybe we simplify to:

```suggestion
# this code runs all workflows in the density folder
```

or maybe remove the run_all_density_workflows and folks can run them individually? 

Not sure.

Reply:
will delete this file and move others to a folder that specifies that they are useful extra code files